### PR TITLE
docs: translate basic/config.md

### PR DESCRIPTION
**Pull request by Brian175**
'config.md' is translated by @jwssdwed and proofread by @Brian175 and finalized by @mmoaay

### CODE REVIEW

**Comment by popomore** [comment]
@Brian175 where are you proofreading the translation?

**Comment by Brian175** [comment]
@popomore hello, this translation is from swiftgg and we have  proofreaded and finalized to some degree to gurantee the quality

File: docs/source/en/basics/config.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+module.exports = appInfo => {
+  return {
+    logger: {
+      dir: path.join(appInfo.baseDir, 'logs'),
+    },
+  };
+};
+```
+The build-in appInfo contains:
+<table>
```
Comment (dead-horse) [inline docs/source/en/basics/config.md]: use markdown syntax

File: docs/source/en/basics/config.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+```
+config
+|- config.default.js
+|- config.test.js
+|- config.prod.js
+|- config.unittest.js
+|- config.local.js
+```
+```config.default.js``` is the default file for configuration, and all environments will load this file. Besides, this is usually used as default configuration file for development environment.
```
Comment (popomore) [inline docs/source/en/basics/config.md]: inline code can change to single `

File: docs/source/en/basics/config.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+Or we can simply set a boolean value
+
+```js
+module.exports = {
+  i18n: false,
+};
+```
+
+### Importing plugins
```
Comment (Brian175) [inline docs/source/en/basics/config.md]: how about 'load' or 'use'

**Comment by popomore** [comment]
cc @atian25

**Comment by popomore** [comment]
https://github.com/eggjs/egg/issues/363

**Pull request merged**